src/rpc/RPCCenter.cpp

```diff
@@ -0,0 +1,102 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2025, the clio developers.
+
+    Permission to use, copy, modify, and distribute this software for any
+    purpose with or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL,  DIRECT,  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include "rpc/RPCCenter.hpp"
+
+#include <string_view>
+#include <unordered_set>
+
+namespace rpc {
+
+namespace {
+
+std::unordered_set<std::string_view> const kHANDLED_RPCS = {
+    "account_channels",
+    "account_currencies",
+    "account_info",
+    "account_lines",
+    "account_nfts",
+    "account_objects",
+    "account_offers",
+    "account_tx",
+    "amm_info",
+    "book_changes",
+    "book_offers",
+    "deposit_authorized",
+    "feature",
+    "gateway_balances",
+    "get_aggregate_price",
+    "ledger",
+    "ledger_data",
+    "ledger_entry",
+    "ledger_index",
+    "ledger_range",
+    "mpt_holders",
+    "nfts_by_issuer",
+    "nft_history",
+    "nft_buy_offers",
```

Comment:
List is not sorted (maybe other one as well).

Is there a way not to hardcode this?